app/assets/src/stylesheets/application.scss

```diff
@@ -1,7 +1,8 @@
 @import "bourbon";
 @import "00-base/base";
 @import "01-atoms/buttons";
+@import "02-molecules/actions";
 @import "02-molecules/forms";
 @import "02-molecules/project-summary";
 @import "03-organisms/header";
-@import "04-templates/templates";
\ No newline at end of file
+@import "04-templates/templates";
```

Comment:
We need to make git enforce line-ending rules pre-commit or something so we don't keep going back and forth on this.